.githooks/pre-push.ps1

```diff
@@ -0,0 +1,29 @@
+# Get the list of changed files
+$ChangedFiles = git status --porcelain | ForEach-Object {
+    ($_ -replace "^\s*\S+\s+", "")  # Handle file paths with possible leading status codes
+}
+
+# Extract modified modules
+$Modules = $ChangedFiles |
+    Where-Object { $_ -like "Commands/*" } |  # Filter files under commands folder
+    ForEach-Object { ($_ -replace "^Commands/", "") } |  # Remove the prefix
+    ForEach-Object { ($_ -split "/")[0] } |  # Extract the moudule
+    Sort-Object -Unique  # Ensure uniqueness
```

Comment:
actually I think we can verify the whole aaz repo. BTW, please add git rebase operation to sync with the upstream repo

Reply:
it will take over 5 minutes, affect the user experience.